static/app/views/eventsV2/table/tableView.tsx

```diff
@@ -373,8 +373,9 @@ class TableView extends React.Component<TableViewProps> {
 
       switch (action) {
         case Actions.TRANSACTION: {
-          const maybeProject = projects.find(project => project.slug === dataRow.project);
-
+          const maybeProject = projects.find(
```

Comment:
Seems like this fixes the case where the project column isn't selected but if project is selected as a column, the "Go to summary" now goes to the performance landing page

Reply:
Hey Shruthi. Hmm, I did test with `project` selected as a column and `eventsV2` does still return the `project.name` value in the response ie "Go to summary" still goes to the correct Transaction Summary page.

Here's the columns I selected to test with:
![image](https://user-images.githubusercontent.com/83961295/152179793-f2316bf3-ca6b-4a0f-ab9c-0ba4ce1f1cbc.png)

let me know if I missed a test case here!

Reply:
Updated tests and updated to support `project` or `project.name`!